bibedit/record_cdfnote.xml

```diff
@@ -0,0 +1,69 @@
+<!-- BibEdit-Template-Name: cdfnote -->
+<!-- BibEdit-Template-Description: CDF internal note -->
+<record>
+  <datafield tag="035" ind1=" " ind2=" ">
+    <subfield code="a">CDFNote-####</subfield>
+    <subfield code="9">CDF</subfield>
+  </datafield>
+  <datafield tag="037" ind1=" " ind2=" ">
+    <subfield code="a">CDF-####</subfield>
```

Comment:
Hi @fschwenn, by putting this CDF-#### place mark as non volatile, isn't there a higher risk of entering them by mistake in the final record?